lib/ranked-model/ranker.rb

```diff
@@ -208,6 +214,11 @@ def rearrange_ranks
         end
       end
 
+      def handle_cannot_rearrange_ranks(message)
+        logger.warn "[RANKED_MODEL] #{message}. Skipping re-ranking of #{instance.model_name.name} #{instance.id}"
+        nil
```

Comment:
returning nil as implemented will skip-reranking

alternatively, could have a strategy

```ruby
fail RankedModel::Unrankable, message, instance
```

Or, since we're in a `before_save` we can  'throw :abort' in the later rails versions, like

```ruby
  errors.add ranker.column, "Could not re-rank columns: #{message}"
  throw :abort
```

Reply:
see #151